pkg/billing/model.go

```diff
@@ -4,6 +4,34 @@ import (
 	"time"
 )
 
+type Event[Self any] interface {
+	AbsoluteEvent | IncrementalEvent
+
+	eventMethods[Self]
```

Comment:
🤮 oh god I just realized what you did.

Reply:
I just figured it out for myself during the meething. I'd split the comment up, and some of the comment here?

Reply:
Added a comment along the lines of "go read the eventMethods docs"